static/src/javascripts/projects/common/modules/experiments/tests/frontend-dotcom-rendering-epic.js

```diff
@@ -42,37 +47,40 @@ const decodeJson = response => response.json();
 
 const products = ['CONTRIBUTION', 'MEMBERSHIP_SUPPORTER'];
 
-const remoteRenderTest = {
-    id: 'RemoteRenderEpicRoundTwo',
+const canRun = (): boolean =>
+    geolocation !== 'US' &&
+    config.get('page').dcrCouldRender &&
+    config.get('tests').dotcomRenderingControl === 'control';
```

Comment:
Reminder that it would be good to verify that this test is correct (I extrapolated from other tests running, but haven't actually seen this one).

Reply:
Here's an example of what the config looks like when you're in the other variant (i.e. seeing DCR):

<img width="559" alt="Screenshot 2020-03-18 at 09 05 50" src="https://user-images.githubusercontent.com/379839/76943990-2a5ae700-68f8-11ea-9e2c-c6e7617292c4.png">

So I think that supports the way we've got it setup right now.

Reply:
🤞🤞